tests/cstor/gtest/test_uzfs.cc

```diff
@@ -3342,13 +3365,20 @@ TEST(MgmtThreadTest, RebuildFailureWrongRebuildState) {
 	EXPECT_EQ(status, ZVOL_OP_STATUS_FAILED);
 }
 
-/* Rebuild success, Single replica success */
+/* Rebuild failed, Single replica */
 TEST(MgmtThreadTest, RebuildFailureSingleReplica) {
 	uzfs_mgmt_conn_t *conn = (uzfs_mgmt_conn_t *)zinfo->mgmt_conn;
 	mgmt_thread_test_case(24);
 	EXPECT_EQ(status, ZVOL_OP_STATUS_FAILED); // errout on healthy replica
 }
 
+/* Rebuild success, Single replica case when volumes are healthy */
+TEST(MgmtThreadTest, RebuildSuccessSingleReplica) {
+	uzfs_mgmt_conn_t *conn = (uzfs_mgmt_conn_t *)zinfo->mgmt_conn;
+	mgmt_thread_test_case(25);
+	EXPECT_EQ(status, ZVOL_OP_STATUS_OK);
```

Comment:
It should be `ZVOL_OP_STATUS_FAILED` since replica status is set to healthy from code only. Can we use this one  https://github.com/openebs/cstor/blob/3f5b702be598e69954708068f3e18e67eb5e27af/tests/cstor/gtest/test_uzfs.cc#L1428?

Reply:
Changes made in this PR will test the status returned for `ZVOL_OPCODE_START_REBUILD` when rebuilding triggers on `Healthy` replica.
`handle_start_rebuild_req` return success when it is a single quorum replica, zinfo->status is Healthy and zv->rebuild_status is `ZVOL_REBUILDING_DONE`.